pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -201,14 +201,35 @@ in
 
   gitlab-markup = attrs: { meta.priority = 1; };
 
-  gitlab-pg_query = attrs: lib.optionalAttrs (attrs.version == "1.3.0") {
+  gitlab-pg_query = attrs: if (attrs.version == "1.3.0") then {
     dontBuild = false;
     postPatch = ''
       sed -i 's;"https://gitlab.com.*";"${fetchurl {
         url = "https://gitlab.com/gitlab-org/libpg_query/-/archive/gitlab-10-1.0.3/libpg_query-gitlab-10-1.0.3.tar.gz";
-        sha256 = "1519x4v6wrk189mjg4hlfah0f7hjy3syg8kk8b6g644gdspzs26j";
+        sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
This… looks weird? Does this just not get evaluated?

Reply:
Oh, right! This isn't evaluated for GitLab 13.8.4. Should we leave it in (and revert to the correct hash) or remove support for building gitlab-pg_query 1.3.0 from gem-config entirely?

Reply:
I'm kind of wondering why this is checking for the version in the first place. If it's needed, it should be applied and if not, it should be dropped IMO. Or can there be multiple versions of `pg_query` in this one gemset?

Reply:
Not sure about one gemset, but this is the global nixpkgs gem-config, so another gemset could use the old version still.

Reply:
I verified that no other gemsets in nixpkgs are using the old version of gitlab-pg_query using nix-review.